pkg/builder/builder.go

```diff
@@ -92,3 +109,29 @@ func (b *Builder) Build(cacheDir string) error {
 
 	return nil
 }
+
+// processLicenseInformationFromCache : gets cached license information by license key and updates the records to be inserted
+func (b *Builder) processLicenseInformationFromCache(license, licenseDir string, licenseMap map[string]string) string {
+	var updatedLicenseList []string
+	// process license information
+	for _, l := range strings.Split(license, "|") {
+		if val, ok := licenseMap[l]; ok {
+			val = strings.TrimSpace(val)
+			updatedLicenseList = append(updatedLicenseList, val)
+		}
+	}
+
+	// precautionary check
+	// return first <licenseStringLimit> characters if license string is too long
+	result := strings.Join(updatedLicenseList, "|")
```

Comment:
@namandf 
I found next case:
<img width="410" alt="изображение" src="https://github.com/aquasecurity/trivy-java-db/assets/91113035/51f9a01f-0a51-4125-87e8-c849b06f5e1b">
<img width="358" alt="изображение" src="https://github.com/aquasecurity/trivy-java-db/assets/91113035/942bb312-f3bc-4403-aeb6-dc927578224c">

Perhaps we can't to use `|`.
Can you take a look?

Reply:
Don't think it's a problem with `|`. 

It's probably because of a missing check where one of the normalized license values is an empty string. Let me add a precautionary check to address it.

Reply:
Can you also check that we have cases, when we have more than 1 values for licenses.

Reply:
Fixed it.  Added a test case as well.

https://github.com/aquasecurity/trivy-java-db/pull/21/commits/e991f76228160463f34d7a4b8e68556c10921464